adam-core/src/main/scala/org/bdgenomics/adam/rdd/features/FeatureParser.scala

```diff
@@ -169,18 +179,28 @@ class GFF3Parser extends FeatureParser {
  * In lieu of a specification, see:
  * https://samtools.github.io/htsjdk/javadoc/htsjdk/htsjdk/samtools/util/IntervalList.html
  */
-class IntervalListParser extends Serializable {
-  def parse(line: String): (Option[SequenceRecord], Option[Feature]) = {
+private[rdd] class IntervalListParser extends Serializable with Logging {
```

Comment:
Let's create an issue to 1) look into using SAM header code when reading IntervalList format and 2) clean up this dbxref key stuff